Packet++/header/TcpReassembly.h

```diff
@@ -208,6 +230,29 @@ class TcpStreamData
 };
 
 
+/**
+ * @struct TcpReassemblyConfiguration
+ * A structure for configuring the TcpReassembly class
+ */
+struct TcpReassemblyConfiguration
+{
+	/** The flag indicating whether to remove the connection data after a connection is closed */
+	bool doNotRemoveConnInfo;
+	/** How long the closed connections will not be cleaned up. The value is expressed in seconds. If the value is set to 0 then TcpReassembly should use the default value */
+	uint32_t closedConnectionDelay;
+	/** The maximum number of items to be cleaned up per one call of purgeClosedConnections. If the value is set to 0 then TcpReassembly should use the default value */
+	uint32_t maxNumToClean;
+
+	/**
+	 * A c'tor for this struct
+	 */
+	TcpReassemblyConfiguration(bool doNotRemoveConnInfo = false, uint32_t closedConnectionDelay = 5, uint32_t maxNumToClean = 5000) :

```

Comment:
do we really want to set `maxNumToClean` default value to 5000? I was thinking more in the area of 10-50.

In a live traffic environment `reassemblePacket()` will be called a lot so if there are peaks of many connections being closed I don't want that to slow down the TCP reassembly process

Reply:
The default has been changed to 30